modules/oe_theme_helper/tests/src/Kernel/Plugin/Field/FieldFormatter/AddressInlineFormatterTest.php

```diff
@@ -29,20 +29,96 @@ protected function setUp(): void {
 
   /**
    * Tests formatting of address.
+   *
+   * @param array $address
+   *   Address in the field format.
+   * @param string $expected
+   *   Expected result.
+   *
+   * @dataProvider addressFieldDataProvider
```

Comment:
Yeah, let's not do a dataprovider here. The theme tests take a very long time as it is. Can you just loop through an array of test data and create the entity, render, assert and delete? I don't think you even need to save to render.

Reply:
Done